### DISCUSSION TITLE
I can't find this shell tool.

### DISCUSSION THREAD

**Discussion by a3246995** [Flow-Launcher/Flow.Launcher#3102] [Q&A]
I can't find this shell tool as Console Emulator, "D:\apps\cmder\vendor\conemu-maximus5\ConEmu64.exe"

**Comment by taooceros** [comment]
> I can't find this shell tool as Console Emulator, "D:\apps\cmder\vendor\conemu-maximus5\ConEmu64.exe"

Could you explain?
